lib/resources/example_json/r4_examples_communication.rb

```diff
@@ -0,0 +1,171 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_COMMUNICATION_ENTRY ||= {
+      'resourceType': 'Communication',
+      'id': '489580643.0.-4.prsnl',
+      'meta': {
+        'versionId': '0',
+        'lastUpdated': '2020-07-16T21:31:35.457Z'
+      },
+      'text': {
+        'status': 'generated',
+        'div': "&lt;div xmlns=\'http://www.w3.org/1999/xhtml\'>&lt;p>&lt;b>Communication&lt;/b>&lt;/p>&lt;/div>"
+      },
+      'extension': [
+        {
+          'valueCodeableConcept': {
+            'coding': [
+              {
+                'system': 'http://hl7.org/fhir/task-status',
+                'code': 'completed',
+                'display': 'Completed'
+              }
+            ],
+            'text': 'Completed'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/email-status'
+        }
+      ],
+      'status': 'completed',
```

Comment:
Add inResponseTo between lines 31 and 32, e.g.
``` 
               "inResponseTo": [
                    {
                        "reference": "Communication/512657279.0.-4.prsnl"
                    }
                ]
```

Reply:
https://github.com/cerner/fhir.cerner.com/pull/607/commits/99c1be98f1d5fe1d1fa6c1cfa9b11d4c798094b9#diff-7b749c1dd8b022a87bedf2a50fba0c2ffa5fa0b910c5d15a1cb40011ba9004deR32